### ISSUE TITLE
Add more detail to the conflict resolution tool

### ISSUE THREAD

**Issue by exhuma** [canton7/SyncTrayzor#507]
When selecting a conflict, you currently only see the filename, last modified date and size.

The size is abbreviated and loses detail for large files. For example, I recently had a file which was shown on both as "3.9MiB". But one of them was corrupted. On closer inspection, the file-size on the corrupted file was a few bytes smaller. But this was hidden in the "3.9MiB" display. It would be nice to see the real bytes value. Or at least make it optional. Especially for the case of conflicts this small difference might be important.

Additionally, showing a quick hash, like a CRC32 or MD5 would be interesting to see. I have some conflicts which have the same byte-content. A CRC32 would be very useful to see in the details pane.

**Comment by Tiim** [comment]
For text files a builtin diff view would be very useful as well.

**Comment by uglygus** [comment]
Yes to including an md5 hash! I frequently notice a handful of conflicts (4-8) and after running md5sum on them all I realize that  there contents are all the same or there are only two distinct hashes. Knowing that upfront would be great. There would probably need to be a file size limit.  4Gb video files can't be hashed in a reasonable time for the UI.
